diffrax/_brownian/tree.py

```diff
@@ -138,18 +175,13 @@ class VirtualBrownianTree(AbstractBrownianPath):
     """Brownian simulation that discretises the interval `[t0, t1]` to tolerance `tol`.
 
     !!! info "Levy Area"
-
-        Can be initialised with `levy_area` set to `diffrax.BrownianIncrement`, or
-        `diffrax.SpaceTimeLevyArea`. If `levy_area=diffrax.SpaceTimeLevyArea`, then it
-        also computes space-time Lévy area `H`. This is an additional source of
-        randomness required for certain stochastic Runge--Kutta solvers; see
-        [`diffrax.AbstractSRK`][] for more information.
-
-        An error will be thrown during tracing if Lévy area is required but is not
-        available.
-
-        The choice here will impact the Brownian path, so even with the same key, the
-        trajectory will be different depending on the value of `levy_area`.
+        The parameter `levy_area` can be set to one of:
+        - [`diffrax.BrownianIncrement`][] (default, generates the increment of W)
+        - [`diffrax.SpaceTimeLevyArea`][] (generates W and the space-time Levy area H)
+        - [`diffrax.SpaceTimeTimeLevyArea`][] (generates W, H and the space-time-time
+                                                Levy area K)
+        The choice of `levy_area` will impact the Brownian path, so even with the same
```

Comment:
And likewise a newline afterwards.

Reply:
Sorry, I really should have known this by now 😅